modules/snooker/src/main/ts/ephox/snooker/resize/BarPositions.ts

```diff
@@ -4,11 +4,13 @@ import { Direction, Height, SugarElement, SugarLocation, Width } from '@ephox/su
 export interface RowInfo {
   readonly row: number;
   readonly y: number;
+  readonly cell: SugarElement<HTMLTableCellElement>;
 }
 
 export interface ColInfo {
   readonly col: number;
   readonly x: number;
+  readonly cell: SugarElement;
```

Comment:
```suggestion
  readonly cell: SugarElement<HTMLTableCellElement>;
```
maybe? Not 100% sure on the types just at a glance though.

Reply:
I think it would be HTMLTableCellElement from looking where BarPositions is used in the code. For now I have reverted the changes on BarPositions as I am not using the cell in my implementation.